src/components/ErrorBar.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+import { useConfig } from '../hooks/useConfig.js'
+import { cn } from '../lib/utils.js'
+import styles from '../styles/ErrorBar.module.css'
```

Comment:
meant to file an issue, but theres currently a bug in the errorbar styles on `space`. hoping this fixes it.

![Screenshot at 2025-03-30 21-30-54](https://github.com/user-attachments/assets/f265f77f-c868-4198-9990-a1f39bffb529)

Reply:
Oh, I'll look at it, or fix it in another PR

Reply:
I think it's fixed, but could you check ? BTW, I updated the space (https://huggingface.co/spaces/hyperparam/hyperparam) so you can try there too

Reply:
Yup fixed thanks!